WordPress/Classes/ViewRelated/Jetpack/Jetpack Scan/View Models/JetpackScanThreatViewModel.swift

```diff
@@ -309,8 +311,128 @@ struct JetpackScanThreatViewModel {
     }
 }
 
+extension JetpackScanThreatViewModel {
+
+    private static func attributedString(for context: JetpackThreatContext?) -> NSAttributedString? {
+        guard let context = context else {
+            return nil
+        }
+
+        let lineFont = UIFont.monospacedSystemFont(ofSize: 13, weight: .semibold)
+        let numberFont = UIFont.monospacedSystemFont(ofSize: 13, weight: .bold)
+        let attrString = NSMutableAttributedString()
+
+        let biggestLen = String(context.lines.last?.lineNumber ?? 0).count
```

Comment:
This name is pretty confusing. We could probably rename it to `lastNumberLength` which is slightly better, then add some documentation for it:

```        
        // Calculates the "longest" number string length
        // this will be used to pad the number column to make sure it's all the same size regardless of
        // the line number length
        //
        // ie: last line number is 1000, which is 4 chars
        // when processing line 50 we'll append 2 spaces to the end so it ends up being equal to 4
```